### ISSUE TITLE
Train loss doesn't converge...

### ISSUE THREAD

**Issue by iseunghoon** [kvsnoufal/Pytorch-FaceNet-DogDataset#3]
loss is not decreased during training... I set hyperparameter same as config file.
